src/bin/zmcontrol

```diff
@@ -559,12 +560,49 @@ sub getHostName {
 }
 
 sub displayVersion {
+    my $zal_path =     "/opt/zextras/lib/ext/carbonio/zal.jar";
```

Comment:
Can we define these two $zal_path and $zextras_path in the global scope to avoid duplication?

Reply:
Done